src/lambda/expression.rs

```diff
@@ -31,11 +32,17 @@ pub enum Expression {
   EqualTo(Box<Expression>, Box<Expression>),
   Unsafe(Unsafe),
```

Comment:
```suggestion
  IO(IO),
```

Rename `Unsafe` to `IO`

Reply:
Move Unsafe to `io.rs`